spec/classes/init_spec.rb

```diff
@@ -56,6 +56,15 @@
         :enable_db_netgroup        => false,
         :enable_opt_auto_propagate => true,
       },
+    'suse13' =>
```

Comment:
Please add the corresponding spec tests for service_provider to check that you can set it to a string and another test that shows that a non-string will trigger a failure.